packages/site/src/designList.ts

```diff
@@ -13,7 +13,7 @@ export const designList = [
   },
   {
     title: "Breakpoints",
-    to: "/design/responsive-breakpoint",
```

Comment:
Doing this to match what's currently in Storybook and the figma (as far as sidenav goes)